### ISSUE TITLE
Build script uses make where it should use ${MAKE}

### ISSUE THREAD

**Issue by ke6jjj** [helium/erlang-stats#2]
# Summary

`blob/master/c_src/build_stats.sh` invokes `make` directly when it ought to use whatever make application is defined in the environment ($MAKE). This is especially important on systems where the default `make` is not GNUMake. (e.g. FreeBSD systems).

# Symptoms

While building `helium/blockchain-etl` on a FreeBSD system, I encountered this set of errors when the build script reached the `erlang_stats` dependency:

```
===> Compiling erl_angry_purple_tiger
===> Compiling erlang_stats
Cloning into 'stats'...
HEAD is now at 659b1d9 version 2.2.0
make[1]: "/home/blockchain-etl/_build/default/lib/erlang_stats/c_src/Makefile" line 18: Missing dependency operator
make[1]: "/home/blockchain-etl/_build/default/lib/erlang_stats/c_src/Makefile" line 23: Missing dependency operator
make[1]: "/home/blockchain-etl/_build/default/lib/erlang_stats/c_src/Makefile" line 24: warning: duplicate script for target "ifeq" ignored
make[1]: "/home/blockchain-etl/_build/default/lib/erlang_stats/c_src/Makefile" line 22: warning: using previous script for "ifeq" defined here
make[1]: "/home/blockchain-etl/_build/default/lib/erlang_stats/c_src/Makefile" line 24: warning: duplicate script for target "(,)" ignored
... 
```

These errors are pretty indicative that BSD Make is trying to parse a GNU Makefile. After modifying `_build/default/lib/erlang_stats/c_src/build_stats.sh` like this:

```
--- a/c_src/build_stats.sh
+++ b/c_src/build_stats.sh
@@ -8,4 +8,4 @@ fi
 # pinned to 2.2.0 commit, no tag
 (cd stats ; git reset --hard 659b1d9af2208de32fa17bcee9e4f3b0c5063a22)
 
-make
+${MAKE}
```

I was able to resume and complete the build normally.

**Comment by ke6jjj** [comment]
I've submitted #3 to fix this issue.

**Comment by evanmcc** [comment]
Merged #3 and will upgrade miner master in the next day or two to include this.
